storm-core/src/jvm/org/apache/storm/command/Rebalance.java

```diff
@@ -57,6 +58,15 @@ public static void main(String[] args) throws Exception {
         }
 
         Map<String, Object> confOverrides = (Map<String, Object>) cl.get("t");
+        Map<String, Object> jvmOpts = Utils.readCommandLineOpts(); // values in -Dstorm.options (originally -c in storm.py)
```

Comment:
This is mostly a question:  Do we want to include default `storm.yaml` into this too? 

for example: 
https://github.com/apache/storm/blob/master/storm-core/src/jvm/org/apache/storm/command/ShellSubmission.java#L35 
this reads `storm.yaml` and commandLineOpts

Reply:
Offline discussion with Bipin :

> I don’t think we want that - that would be a reset+. Since rebalance should an incremental change to the currently running Topo.